vm/src/builtins/weakproxy.rs

```diff
@@ -66,6 +67,24 @@ impl PyWeakProxy {
             None => Err(new_reference_error(vm)),
         }
     }
+
+    fn len(&self, vm: &VirtualMachine) -> PyResult<usize> {
+        match self.weak.upgrade() {
+            Some(obj) => obj.length(vm),
+            None => Err(new_reference_error(vm)),
+        }
+    }
+
+    #[pymethod(magic)]
+    fn contains(&self, needle: PyObjectRef, vm: &VirtualMachine) -> PyResult<bool> {
+        match self.weak.upgrade() {
+            Some(obj) => {
+                // let seq = PySequence::try_protocol(obj.as_ref(), vm)?;
```

Comment:
Maybe mistake? Is this comment intended?

Reply:
It's a mistake! 😅